css/style.css

```diff
@@ -32,8 +58,60 @@ header a {
   transition: all 0.5s ease-out;
 }
 
-.nav-header {
-  display: none;
+.menu {
+  display: flex;
+  align-self: center;
+  align-content: flex-end;
+  flex-direction: row;
+  justify-self: flex-end;
+}
+
+.menu-display {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  width: 100%;
+  background-color: #3c3a39;
+  color: #fff5e1;
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
  - Please make sure the mobile display is responsive by removing the while space on the right.
  - Please make sure the mobile menu looks exactly like the Figma design.

| Your Design |Figma Design |
| :---:   | :---: | 
|![Screenshot from 2023-02-08 15-36-48](https://user-images.githubusercontent.com/92303891/217576716-5c70a7dc-df96-4a85-a301-e5d0de6b6ace.png)| ![Screenshot from 2023-02-08 15-30-42](https://user-images.githubusercontent.com/92303891/217575472-0b95615a-6766-4ad5-99a2-2bcbda255ec8.png) |